packages/nodes-base/nodes/Kafka/Kafka.node.ts

```diff
@@ -74,6 +76,43 @@ export class Kafka implements INodeType {
 				type: 'boolean',
 				default: false,
 			},
+			{
+				displayName: 'Use Schema Registry',
+				name: 'useSchemaRegistry',
+				type: 'boolean',
+				default: false,
+				description: 'Use Confluent Schema Registry.',
+			},
+			{
+				displayName: 'Schema Registry URL',
+				name: 'schemaRegistryUrl',
+				type: 'string',
+				required: true,
+				displayOptions: {
+					show: {
+						useSchemaRegistry: [
+							true,
+						],
+					},
+				},
+				default: 'https://schema-registry-domain:8081',
```

Comment:
Maybe using the placeholder instead of default is more adequate (because this default URL does not resolve), the same for Kafka Trigger.

Reply:
done